### PR TITLE
Added Disclaimer Page

### LINKED ISSUE #160: Need to add a disclaimer page to the application
## Description
As an organization hosting the site,
we need to have a disclaimer page,
so that we can add our source references and avoid any copyright issues if any.

### Update [Required]
- [ ] New page/screen that contains the disclaimer details

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentors.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by lunayach**
### Description
Added a Disclaimer page that tells about PC-Prep-Kit not being responsible for any crept error. The page also lists application sources.

Fixes #160 

### Type of Change:

- Code
- Quality Assurance
- Outreach

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
All new/old Karma and Mocha tests passed.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: src/assets/languages/english.json [hunk 1]
```diff
@@ -119,6 +119,11 @@
       "dashboardLink": "Go to Dashboard",
       "badgeLabel": "Expert"
     },
+    "disclaimer": {
+      "heading": "DISCLAIMER",
+      "body": "It has been tried best to ensure that the application is error free. If in any case, some error is found, PC-Prep-Kit shall not be made responsible. Please report such errors via GitHub issues or slack channel. ",
```
Comment (Raxerz) [inline src/assets/languages/english.json]: Change the body to the following:

"In accord with our non-profit mission, the information - text, images and everything else present on this site has been researched and gathered from various legitimate sources and is being used purely for informational purposes only. 

The objective of this site is to promote malaria awareness through some exciting educational activities and games and was not intended to serve as a replacement for professional medical advice, diagnosis or treatment. Always seek the advice, diagnosis and/or treatment from qualified doctors for any malaria related help. 

If the imformation provided on this site is incorrect, misleading, invalid or outdated, please notify us on our Github repository <insert slack url as hyperlink for 'Github repository'> or on our slack channel <insert slack url as hyperlink for 'slack channel'>."

@lunayach does this sound good. Please suggest any changes that are necessary.

@Buddhiprabha @shreyans29 @paavininanda does this look fine? Please feel free to suggest any changes.

File: src/assets/languages/english.json [hunk 2]
```diff
@@ -119,6 +119,11 @@
       "dashboardLink": "Go to Dashboard",
       "badgeLabel": "Expert"
     },
+    "disclaimer": {
+      "heading": "DISCLAIMER",
```
Comment (Raxerz) [inline src/assets/languages/english.json]: Change the heading to "Disclaimer and Free use policy"

File: src/assets/languages/english.json (hunk 1 above)
Comment (paavininanda) [reply src/assets/languages/english.json]: Yes this looks good @Raxerz

Comment (Buddhiprabha) [reply src/assets/languages/english.json]: This is very good, please do the changes @lunayach

**Comment by lunayach** [comment]
Addressed the comment.

**Pull request merged**